gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/util/NumberHelperTest.java

```diff
@@ -96,6 +98,21 @@ public class NumberHelperTest {
                     new Quartet<>(BigDecimal.ONE, BigDecimal.ONE, BigDecimal.class, BigDecimal.class)
             );
 
+    private final static List<Triplet<Number, Number, String>> OVERFLOW_CASES = Arrays.asList(
```

Comment:
Very minor nit: This doesn't need to be done, but ParameterizedTest could have been used here to split each into individual test.